src/unistd.rs

```diff
@@ -214,6 +214,36 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+pub enum Whence {
+    SeekSet,
+    SeekCur,
+    SeekEnd
+}
+
+impl Whence {
+    fn to_libc_type(&self) -> c_int {
+        match self {
+            &Whence::SeekSet => 0,
+            &Whence::SeekCur => 1,
+            &Whence::SeekEnd => 2
```

Comment:
Should I use those constants directly ? What made me not do it in the first place was that the type is too permissive, any c_int could be sent; and that I wan't sure how much of libc's types should be in nix's API.